### PR TITLE
app-example-config tutorial

### PR DESCRIPTION
**Pull request by seancdavis**
New things here:

- [Tutorials section](https://www-grouparoo-com-git-177550330-app-example-config-grouparoo.vercel.app/docs/tutorials) in the docs.
- [Tutorial for SQLite -> Mailchimp](https://www-grouparoo-com-git-177550330-app-example-config-grouparoo.vercel.app/docs/tutorials/app-example-config) to support code in [app-example-config](https://github.com/grouparoo/app-example-config).
- Flexibility to be able to pass the headings you want to appear in the in-page nav. There was a lot of content here, and I wanted to break it up with headings, which also helps with SEO juice. But it made the in-page nav longer than it had space for and pretty tough to read.
- Fix for duplicate keys on in-page nav items.

### CODE REVIEW

File: pages/docs/tutorials/app-example-config.mdx [hunk 1]
```diff
@@ -0,0 +1,672 @@
...
+
+    $ cd /path/to/workspace
+    $ grouparoo init app-example-config
+    $ cd app-example-config
+
+_Note: If the directory for your project exists, you can run `grouparoo init .` and Grouparoo will create the project inside that directory._
+
+## Step 3: Add Source Database
+
+We have some seed data for you to work with for this example. You can download the database directly [here](/downloads/sqlite-source.sqlite). Place that file anywhere on your file system that your current user can read.
```
Comment (evantahler) [inline pages/docs/tutorials/app-example-config.mdx]: ... This SQLite database contains the contents of this CSV [link], which is a prototypical ecommerce example.

File: pages/docs/tutorials/app-example-config.mdx [hunk 2]
```diff
@@ -0,0 +1,672 @@
...
+    $ grouparoo validate
+
+    🦘 Grouparoo: validate
+
+    Validating 1 objects...
+    info: [ config ] validated App `product_db` (product_db)
+
+    ✅ Validation succeeded - 1 config objects OK!
+
+Hooray! When you're ready let's [apply those changes](/docs/cli/config#apply), which means we're saving them to the database.
```
Comment (evantahler) [inline pages/docs/tutorials/app-example-config.mdx:185]: I've been trying to use the phrase `Grouparoo database` to disambiguate our internal DB from any sources or destinations.

File: pages/docs/tutorials/app-example-config.mdx [hunk 3]
```diff
@@ -0,0 +1,672 @@
...
+    Applying 1 objects...
+    notice: [ config ] created App `product_db` (product_db)
+
+    ✅ Config applied - 1 config objects up-to-date!
+
+Now we have an app and we can add some Sources!
+
+## Step 5: Add Users Source and Properties
+
+Next, it's time to add our Sources and their Properties. A Source is the place your customer data lives. In this case, we're mocking a local data warehouse type of setup. Properties are attributes for your Profiles (in Grouparoo, we refer to users as _Profiles_).
```
Comment (evantahler) [inline pages/docs/tutorials/app-example-config.mdx]: No idea if "local datawarehouse" is a good metaphor or confusing to folks that already have one.

File: pages/docs/tutorials/app-example-config.mdx [hunk 4]
```diff
@@ -0,0 +1,672 @@
...
+
+We are now at the point where we have _essentially_ everything included in [the app-example-config project](https://github.com/grouparoo/app-example-config). The only thing we're missing is writing any tests to add confidence to the data you're exporting. More on that below.
+
+You're ready to run Grouparoo! Do it!
+
+    $ grouparoo start
+
+This will begin the process of importing data from SQLite into Grouparoo and exporting it to Mailchimp.
+
+After you run the command the server will go wild, and that's a good thing. But it's tough to follow. You can open your browser and run the web app for some visibility and observability into what's happening under the hood. The app is available at [localhost:3000](http://localhost:3000/).
```
Comment (evantahler) [inline pages/docs/tutorials/app-example-config.mdx]: I *think* google dings you for linking to localhost/a URL that doesn't resolve.  May not a link here.

File: components/pageNavigation.tsx [hunk 5]
```diff
@@ -21,7 +23,7 @@ export function PageNavigation({
 
   function buildPageNavigation() {
     const headerElements = globalThis.document.querySelectorAll(
-      "h1, h2, h3, h4, h5, h6"
+      pageNavSelector || "h1, h2, h3, h4, h5, h6"
```
Comment (evantahler) [inline components/pageNavigation.tsx:26]: Fancy!

**Review by evantahler — APPROVED** [review]
This is great!  

None of my comments below or here are blocking.

Some thoughts:
* Perhaps the CTA to "see the code" (link to the Github repo) should be more prominent, and repeated at the top and bottom
* We could probably use a header graphic, even a `[sqllite logo] -> [mailchimp logo]`

**Comment by seancdavis** [comment]
Note: This also brought an SEO component to make it easier to add SEO meta tags to page templates.

**Pull request merged**